### ISSUE TITLE
Mac OS Voiceover in Safari says Visualization is Empty

### ISSUE THREAD

**Issue by cgg-cmu** [cmu-delphi/www-covidcast#528]
**Actual Behavior:**

When focus goes to the visualization map the screen reader Voiceover in Safari says that "You are currently in a empty visualization" 

When I...

Focus on the map while using Voiceover.

1. go to https://cms-staging.andrew.cmu.edu/covid/?sensor=doctor-visits-smoothed_adj_cli&level=county&date=20200701&signalType=value&encoding=color&mode=overview
2. Activate Voiceover (if not already activated)
3. Tab to map in page

**Expected behavior**

The screenreader should be able to read the current state of the map (eg options selected)

**Screenshots**

![image](https://user-images.githubusercontent.com/60197405/94747817-d53f6980-034d-11eb-9652-3d82563874b3.png)

**Rating Scale (1 is minor, 7 is severe):**

5 

**Context**

- Browser: Safari Mac OS
